entity/scenario_intersection/include/scenario_intersection/intersection.h

```diff
@@ -66,40 +68,40 @@ class Intersection
         }
       }
 
-      bool changeColor(ScenarioAPI& simulator) const
-      {
-        if (target_ < 0 or color_ == Color::Blank)
-        {
-          return simulator.resetTrafficLightColor(target_, false);
-        }
-        else // NOTE: Maybe specified illiegal traffic-light-id.
-        {
-          return simulator.setTrafficLightColor(target_, boost::lexical_cast<std::string>(color_), false);
-        }
-      }
-
-      bool changeArrow(ScenarioAPI& simulator) const
-      {
-        simulator.resetTrafficLightArrow(target_, false);
-
-        if (0 <= target_)
-        {
-          return
-            std::all_of(
-              std::begin(arrows_), std::end(arrows_),
-              [&](const auto& each)
-              {
-                return
-                  simulator.setTrafficLightArrow(
-                    target_, boost::lexical_cast<std::string>(each), false);
-              });
-        }
-      }
-
-      bool operator()(ScenarioAPI& simulator) const
-      {
-        return changeColor(simulator) and changeArrow(simulator);
-      }
+      // bool changeColor(ScenarioAPI& simulator) const
```

Comment:
Why is this code commented?

Reply:
It has been commented out as the `ScenarioAPI` class is currently not available. Once those changes are merged this branch can be rebased and references to `ScenarioAPI` can be add back.